### DISCUSSION TITLE
layouts within a sub-directory

### DISCUSSION THREAD

**Discussion by sxshateri** [livewire/livewire#3808] [1. Help]
I have different layouts for my admin panel which is located at ```layouts/admin``` and I tried to use dot prefix to declare the layout inside the component like ```->layout('layouts.admin.app')``` but it keeps son pulling the main layout which is located at the root of the ```layout``` directory instead of one inside the ```layouts/admin```. why is that and how to fix it?

**Comment by joshhanley** [comment]
@sxshateri I've just tested this locally and it's working ok for me.

Can you share your render method and also what your folder structure looks like?

This is my test

```php
<?php

namespace App\Http\Livewire;

use Livewire\Component;

class Main extends Component
{
    public function render()
    {
        return view('livewire.main')
            ->layout('layouts.admin.app');
    }
}
```

My normal layout is stored in `resources/views/layouts/app.blade.php`.
But this admin one that I tested and works is stored in `resources/views/layouts/admin/app.blade.php`.

Hope this helps!
